easybuild/easyconfigs/p/PileOMeth/PileOMeth-0.1.11.eb

```diff
@@ -0,0 +1,40 @@
+##
+#  This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+#  Homepage: https://www.adelaide.edu.au/phoenix/
+# 
+#  Copyright::   adelaide.edu.au/phoenix
+#  Authors:: Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+#  License::
+#
+#  Notes::
+###
+
+easyblock = 'MakeCp'
+
+name = 'PileOMeth'
+version = '0.1.11'
+
+homepage = 'https://github.com/dpryan79/PileOMeth'
+description = """PileOMeth processes a coordinate-sorted and indexed BAM or CRAM file containing some form of BS-seq alignments. 
+PileOMeth extracts per-base methylation metrics from them. 
+PileOMeth requires an indexed fasta file containing the reference genome as well.
+"""
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(version)s.zip']
+source_urls = ['https://github.com/dpryan79/PileOMeth/archive/']
+
+dependencies = [
+    ('HTSlib', '1.2.1', '', ('foss', '2017a'))
```

Comment:
@boegel thanks. Is that a convention to use a tool chain option if any dependencies use one, even when parent itself not use any?